src/modules/jcms_rest/src/Plugin/rest/resource/AbstractRestResourceBase.php

```diff
@@ -322,6 +324,22 @@ protected function filterPageAndOrder(QueryInterface &$query, $sort_by = NULL) {
   }
 
   /**
+   * Apply filter for date range by amending query.
+   *
+   * @param \Drupal\Core\Entity\Query\QueryInterface $query
+   */
+  protected function filterDateRange(QueryInterface &$query) {
+    $start_date = $this->getRequestOption('start-date');
+    if (!empty($start_date)) {
+      $query->condition('created', strtotime($start_date), '>=');
```

Comment:
My validation/date creation comment in https://github.com/elifesciences/search/pull/103 apply here too.